apollo-client/src/main/java/com/ctrip/framework/apollo/internals/RemoteConfigRepository.java

```diff
@@ -255,6 +259,19 @@ private ApolloConfig loadApolloConfig() {
 
           ApolloConfig result = response.getBody();
 
+          if (result != null) {
+            ConfigSyncType configSyncType = ConfigSyncType.fromString(result.getConfigSyncType());
+
+            if (configSyncType == ConfigSyncType.INCREMENTAL_SYNC) {
```

Comment:
What actions should we take when the config sync type is `UNKNOWN`?

Reply:
throw ApolloConfigException